Gamma_Analysis.py

```diff
@@ -167,63 +168,35 @@ def peak_finder(spectrum, energy):
 def peak_measurement(M, energy, sub_regions='both'):
```

Comment:
As discussed by email, ROI_Maker will handle sub_regions so it is not needed here.